src/app/clusters/scenes-server/SceneTableImpl.cpp

```diff
@@ -68,15 +67,15 @@ DefaultSceneHandlerImpl::SerializeAdd(EndpointId endpoint,
         pairCount++;
     }
     ReturnErrorOnFailure(pair_iterator.GetStatus());
-    app::DataModel::List<app::Clusters::Scenes::Structs::AttributeValuePair::Type> attributeValueList(aVPairs, pairCount);
+    app::DataModel::List<AttributeValuePair::Type> attributeValueList(aVPairs, pairCount);
 
     return EncodeAttributeValueList(attributeValueList, serializedBytes);
 }
 
 CHIP_ERROR DefaultSceneHandlerImpl::Deserialize(EndpointId endpoint, ClusterId cluster, const ByteSpan & serializedBytes,
                                                 app::Clusters::Scenes::Structs::ExtensionFieldSet::Type & extensionFieldSet)
```

Comment:
this is another example where a using ExtensionFieldSet = ...
Would be beneficial.

Note, only add using to the .h if it is beneficial to have it there (some types might only be in the .cpp file)